### PR TITLE
[ID-47]Fix nudges processes docs

### LINKED ISSUE #47: Fix nudges processes docs

### PR DESCRIPTION
**Pull request by stefanvit**
resolved #47

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Fix nudges processes docs [#47](https://github.com/rokwire/lms-building-block/issues/47)
```
Comment (petyos) [inline CHANGELOG.md:8]: It should be in Fixed section.

File: driver/web/docs/schemas/nudges/NudgesProcess.yaml [hunk 2]
```diff
@@ -10,14 +12,26 @@ properties:
...
     type: object
-  items: 
-    type: object
+    properties:
+      number:
+       type: integer
+      items:
+        type: object
+        properties: 
+          netID:
```
Comment (petyos) [inline driver/web/docs/schemas/nudges/NudgesProcess.yaml:31]: All should follow the conventions net_id instead of netID.

**Review by petyos — CHANGES_REQUESTED** [review]

**Review by petyos — APPROVED** [review]
LGTM

**Comment by petyos** [comment]
Hi @stefanvit , could you please fix the conflicts and merge this on develop? Thanks.

**Comment by stefanvit** [comment]
> Hi @stefanvit , could you please fix the conflicts and merge this on develop? Thanks.

Done

File: driver/web/docs/schemas/nudges/NudgesProcess.yaml [hunk 3]
```diff
@@ -1,6 +1,8 @@
 required:
   - id
   - mode
+  - createdAt
```
Comment (petyos) [inline driver/web/docs/schemas/nudges/NudgesProcess.yaml:4]: Should not be created_at and completed_at?

File: driver/web/docs/schemas/nudges/NudgesProcess.yaml [hunk 4]
```diff
@@ -10,14 +12,26 @@ properties:
     type: string
   mode:
     type: string
+  createdAt:
```
Comment (petyos) [inline driver/web/docs/schemas/nudges/NudgesProcess.yaml:15]: Should not be created_at and completed_at?

**Pull request merged**